src/_lib/sass/styles.scss

```diff
@@ -83,6 +83,7 @@
 @import "src/navs";
 @import "src/scaffolding";
 @import "src/popovers";
+@import "src/feature-alerts";
```

Comment:
I see this as more of an addition than a replacement

Reply:
It is a small addition that holds the width of the btn.

Reply:
ok, I see what you mean, I moved it down.